hr_attendance_calendar/models/hr_attendance_day.py

```diff
@@ -139,43 +116,43 @@ def _compute_due_hours(self):
                     if leave.state not in ['validate', 'validate1']:
                         continue
                     if not leave.holiday_status_id.remove_from_due_hours:
-                        # todo: use config instead of 8.0
+                        working_day = float(self.env[
```

Comment:
Here you could make use of a getter in your model `hr.attendance.settings`, but this is working as well.